# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../musicbrainz-trm/gcc43.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- trm-0.2.1/mbid3.cpp.vanilla	2009-02-21 18:32:16.000000000 +0100
+++ trm-0.2.1/mbid3.cpp	2009-02-21 18:32:38.000000000 +0100
@@ -6,6 +6,7 @@
 #ifdef HAVE_LIBID3
 
 #include <stdio.h>
+#include <string.h>
 #include "mbid3.h"
 
 #define DB Debug_v("%s:%d", __FILE__, __LINE__);
